dirac.cfg

```
@@ -852,6 +852,162 @@ Resources
+  Sites
+  {
+    LCG
+    {
+      ##BEGIN SiteConfiguration
+      LCG.CERN.ch
+      {
+        # Local Storages
+        SE = CERN-RAW, CERN-RDST, CERN-USER # (Optional) SEs Local to the site
+
+        # Overwrite definities of StorageElement (discouraged)
+        # or StorageElementGroups for that Site
+        AssociatedSEs
+        {
+          # Tier1-Failover is now only CERN-FAILOVER when running a Job at CERN
+          Tier1-Failover = CERN-FAILOVER
+        }
+        CE = ce503.cern.ch, ce504.cern.ch # List of CEs for th esite (updated by BDII2CSAgent)
+        Name = CERN-PROD # (Optional) Name of the site from the admin, e.g in GOCDB
+        Coordinates = 06.0458:46.2325 # (Optional) Geographical coordinates
+        Mail = grid-cern-prod-admins@cern.ch # (Optional) Geographical coordinates
+        MoUTierLevel = 0 # (Optional) Tier level
+        Description = CERN European Organization for Nuclear Research # (Optional) ...
+        # Subsection to describe each CE available
+        CEs
+        {
+          # Subsection named as the CE fully qualified name
+          ce503.cern.ch
+          {
+            # Worker node temporary directory
+            wnTmpDir = .
+
+            # CE architecture
+            architecture = x86_64
+
+            # CE operating system in a DIRAC format
+            OS = ScientificCERNSLC_Carbon_6.4
+
+            # Boolean attributes than indicates if the site accept pilots
+            Pilot = False
+
+            # Type of CE, can take values as LCG or CREAM
+            CEType = HTCondorCE
+
+            # Type of 'Inner' CE, normally empty. Default = "InProcess".
+            # Possibilities: potentially all CE types, but in practice
+            # the most valid would be: InProcess, Sudo, Singularity, Pool.
+            # Pool CE in turn uses InProcess (Default)
+            # or Sudo or Singularity. To specify, use Pool/ce_type.
+            # This option can also go at the Queue level.
+            LocalCEType = Pool
+
+            # If the CE is a cream CE the mode of submission
+            SubmissionMode = Direct
+
+            # Number of available processors on worker nodes
+            NumberOfProcessors = 12
+
+            # Number of available worker nodes per allocation.
+            # Values can be a number (e.g. 2 nodes) or a range of values
+            # (e.g. from 2 to 4 nodes) which lets the choice to the batch
+            # system.
+            NumberOfNodes = 2
+            # NumberOfNodes = 2-4
+
+            # CE allows *whole node* jobs
+            WholeNode = True
+
+            # List of tags specific for the CE
+            Tag = GPU, 96RAM
+
+            # List of required tags that a job to be eligible must have
+            RequiredTag = GPU,96RAM
+
+            # Queues available for this VO in the CE
+            Queues
+            {
+              # Name of the queue exactly how is published
```

Actually this is very rarely how it is published, mostly how we (I) massage it in the bdii2csagent

```suggestion
              # Name of the queue exactly how it is published
```